Ds3/Runtime/HttpHelper.cs

```diff
@@ -60,7 +70,7 @@ public static string PercentEncodePath(string path)
         /// <returns></returns>
         public static string PercentEncodeParam(string path)
         {
-            return CustomPercentEscaper.PercentEncode(path, UnreservedCharsParam);
```

Comment:
Query parameters can also contain "?"

Reply:
you are correct, but here we encode the parm itself and not the query
`param          | *<pchar excluding ";"> `